extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -56,9 +56,11 @@ export async function promptImportInternetDatabase(
       );
       commands.executeCommand('codeQLDatabases.focus');
```

Comment:
Thanks for the PR!

I notice this line right above that focuses the databases panel is also unconditionally being called regardless of whether we actually have a non-`undefined` value of `item` as well. Could you move the `showAndLogInformationMessage` up here and include both it and the `codeQLDatabases.focus` in the conditional?

Reply:
:+1: Fixed. Please take another look.

Looks like there's opportunity for a refactor since there's some duplicated code, but I think we can leave that for another PR.